ush/python/pygfs/task/land_analysis.py

```diff
@@ -336,6 +339,15 @@ def finalize(self) -> None:
         statfile = os.path.join(self.task_config.COM_LAND_ANALYSIS, f"{self.task_config.APREFIX}landstat.tgz")
         self.tgz_diags(statfile, self.task_config.DATA)
 
+        logger.info("Copy full YAML to COM")
+        src = os.path.join(self.task_config['DATA'], f"{self.task_config['CDUMP']}.t{self.runtime_config['cyc']:02d}z.letkfoi.yaml")
```

Comment:
```suggestion
        src = os.path.join(self.task_config['DATA'], f"{self.task_config['RUN']}.t{self.runtime_config['cyc']:02d}z.letkfoi.yaml")
```
Please don't use `CDUMP` anymore.

Reply:
Also, isn't this already defined as `self.task_config.APREFIX`?   Why do we need to reconstruct that again here?

Reply:
Thanks @aerorahul for quick review. Good catch, and I made the changes as suggested.